src/main/java/org/pac4j/javalin/CallbackHandler.java

```diff
@@ -36,19 +32,20 @@ public CallbackHandler(Config config, String defaultUrl, Boolean renewSession) {
 
     @Override
     public void handle(@NotNull Context javalinCtx) {
-        final SessionStoreFactory sessionStoreFactory = FindBest.sessionStoreFactory(null, config, JEESessionStoreFactory.INSTANCE);
-        final SessionStore sessionStore = sessionStoreFactory.newSessionStore(javalinCtx);
-        final HttpActionAdapter bestAdapter = FindBest.httpActionAdapter(null, config, JavalinHttpActionAdapter.INSTANCE);
-        final CallbackLogic bestCallbackLogic = FindBest.callbackLogic(null, config, DefaultCallbackLogic.INSTANCE);
-
-        JavalinWebContext context = new JavalinWebContext(javalinCtx);
-        bestCallbackLogic.perform(context,
-                sessionStore,
+        final CallbackLogic callbackLogic;
```

Comment:
No. You should use: https://github.com/pac4j/jee-pac4j/blob/master/jakartaee-pac4j/src/main/java/org/pac4j/jee/filter/CallbackFilter.java#L61

Reply:
aaah... done